gpkitmodels/__init__.py

```diff
@@ -0,0 +1,2 @@
+from gpkit import Variable
+g = Variable("g", 9.81, "m/s^2", "gravitational acceleration")
```

Comment:
add "near the earth's surface" ?

Reply:
hmm or just "earth surface?"

Reply:
sounds good to me!

Reply:
I pushed directly to master.  Sorry I should have asked you to review this again before merging...